src/stories/containers/users/delete-account/delete-account.tsx

```diff
@@ -68,7 +69,11 @@ const DeleteAccount: React.FC<{ username?: string }> = ({ username }) => {
           }}
         >
           <Label isLight={isLight}>{`${
-            isAdmin ? 'Enter your admin password to delete the account' : 'Enter password to delete the account'
+            isAdmin && !isAdminRouter
+              ? 'Enter your admin password to delete the account'
+              : isAdmin && isAdminRouter
+              ? 'Enter password to delete the account'
+              : 'Enter password to delete the account'
```

Comment:
Please, note that this is redundant as the message in both cases is the same one

Reply:
You are right, I will remove